src/main/java/com/work/management/entity/Team.java

```diff
@@ -0,0 +1,46 @@
+package com.work.management.entity;
+
+import com.google.common.collect.ImmutableList;
+import com.work.management.converter.ListConverter;
+import lombok.*;
+import javax.persistence.*;
+import java.util.Date;
+import java.util.List;
+
+
+@Data
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+@Entity
+@Table(name = "TEAMS")
+public final class Team {
+
+    @Column(name = "name")
+    private String name;
+
+    @Id
+    @Column(name = "id")
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private String id;
+
+    @Column(name = "manager")
+    private Integer manager;
+
+    @Column(name = "createdAtTimeStamp")
+    private Date createdAtTimeStamp;
+
+    @Column(name = "lastUpdatedTimeStamp")
+    private Date lastUpdatedTimeStamp;
+
```

Comment:
Use @CreatedDate and @LastModifiedDate on createdAtTimeStamp and lastUpdatedTimeStamp respectively.

https://stackoverflow.com/questions/49954812/how-can-you-make-a-created-at-column-generate-the-creation-date-time-automatical